tools/utils/gh.py

```diff
@@ -40,15 +40,15 @@ def download_onedir_artifact(
             )
             return ExitCode.FAIL
     else:
-        if arch == "arm64":
-            ctx.info(f"Turning passed arch {arch!r} into 'aarch64'")
-            arch = "aarch64"
+        if arch == "aarch64":
+            ctx.info(f"Turning passed arch {arch!r} into 'arm64'")
+            arch = "arm64"
         elif arch == "x64":
             ctx.info(f"Turning passed arch {arch!r} into 'x86_64'")
             arch = "x86_64"
-        if arch not in ("x86_64", "aarch64"):
+        if arch not in ("x86_64", "arm64"):
```

Comment:
if arch not in ("x86_64", "aarch64", "arm64")

Reply:
We don't care about `aarch64` in this context.